__tests__/helpers/locale/language.test.ts

```diff
@@ -45,9 +46,31 @@ describe('Language Tests', () => {
             const result = language("settings", "xx");
             expect(result).toBe("Settings");
         });
-        it('Should return the string key if the key is not found.', () => {
+        it('Should return empty string if the key is not found.', () => {
             const result = language("no_such_string", "xx");
-            expect(result).toBe("no_such_string");
+            expect(result).toBe("");
         });
     });
+
+    describe('supportedLocales', () => {
+        it('Should have all strings in all locales', () => {
+            const deliberatelyBlankStrings = 
```

Comment:
I'm seeing an error below because the key type for this const is inferred as `any` and the key being used to index it below is a `string`.  This should fix that.

`const deliberatelyBlankStrings: { [key: string]: string[] } = `